### PR TITLE
[TASK] Use the vendor folder for Composer packages

### PR DESCRIPTION
**Pull request by brotkrueml**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,9 +10,9 @@ FROM php:8.2-cli-alpine
 COPY . /opt/guides
 WORKDIR /opt/guides
 
-COPY --from=Builder /opt/guides/.Build /opt/guides/.Build
-RUN cp guides.xml .Build/guides.xml
+COPY --from=Builder /opt/guides/vendor /opt/guides/vendor
+RUN cp guides.xml guides.xml
```
Comment (brotkrueml) [inline Dockerfile:14]: @jaapio Is the the correct path then? `.Build/guides.xml` -> `guides.xml`

**Review by linawolf — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (linawolf) [reply Dockerfile:14]: yes that is correct

Comment (linawolf) [reply Dockerfile:14]: however we do not have to copy it then. The guides always searches in the parent directory of vendor

Comment (linawolf) [inline Dockerfile:14]:
```suggestion
```
This line does not make sense, lets remove it

Comment (jaapio) [reply Dockerfile:14]: Yeah, we can simply remove this cp command

Comment (brotkrueml) [reply Dockerfile]: Done

**Pull request merged**